bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/image/BackgroundImageLoader.scala

```diff
@@ -76,12 +77,22 @@ object BackgroundImageLoader {
 
   /** Called when a program is created to download its images */
   def watch(prog: ISPProgram): Unit = {
+    // At startup only load images for active programs
+    def needsImage(ctx: TpeContext): Boolean = {
+      ctx.obsShell.exists(ObservationStatus.computeFor(_) match {
+        case ObservationStatus.INACTIVE | ObservationStatus.OBSERVED => false
+        case _                                                       => true
+      })
+    }
+
     // Listen for future changes
     prog.addCompositeChangeListener(ChangeListener)
     val targets = for {
-        p   <- prog.getAllObservations.asScala.toList
-        obs <- p.getObsComponents.asScala
-        i   <- requestedImage(TpeContext(obs))
+        p      <- prog.getAllObservations.asScala.toList
```

Comment:
There is a bit of magic that does this exactly in `RichProgram`, if you want to import it.  Not a bit deal.